### PR TITLE
VIA Support: Gray Studio HB85

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Add VIA support to the Gray Studio HB85
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/gray_studio/hb85/keymaps/via/config.h [hunk 1]
```diff
@@ -0,0 +1 @@
+#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```
Comment (zvecr) [inline keyboards/gray_studio/hb85/keymaps/via/config.h:1]:
```suggestion
#pragma once

#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```

File: keyboards/gray_studio/hb85/keymaps/via/rules.mk [hunk 2]
```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+
+MOUSEKEY_ENABLE = no
```
Comment (zvecr) [inline keyboards/gray_studio/hb85/keymaps/via/rules.mk:4]: Is this required now we have a 3rd endpoint?

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/gray_studio/hb85/keymaps/via/rules.mk (hunk 2 above)
Comment (mechmerlin) [reply keyboards/gray_studio/hb85/keymaps/via/rules.mk]: You are correct!

File: keyboards/gray_studio/hb85/keymaps/via/config.h [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#pragma once
```
Comment (noroadsleft) [inline keyboards/gray_studio/hb85/keymaps/via/config.h:1]: Can we get a license header here, please?

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/gray_studio/hb85/keymaps/via/config.h (hunk 3 above)
Comment (mechmerlin) [reply keyboards/gray_studio/hb85/keymaps/via/config.h:18]: done

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**